digdag-core/src/main/java/io/digdag/core/database/DatabaseProjectStoreManager.java

```diff
@@ -571,13 +582,15 @@ public void deleteSchedules(int projId)
                 " join projects proj on a.project_id = proj.id" +
                 " join workflow_configs wc on wc.id = wd.config_id" +
                 " where wd.id \\> :lastId" +
+                " and wd.name like :name" +
                 " and <acFilter>" +
                 " order by wd.id" +
                 " limit :limit")
         List<StoredWorkflowDefinitionWithProject> getLatestActiveWorkflowDefinitions(
                 @Bind("siteId") int siteId,
                 @Bind("limit") int limit,
                 @Bind("lastId") long lastId,
+                @Bind("name") String name,
```

Comment:
`namePattern` or something is more correct?

Reply:
renamed at 601788e

Reply:
Hmm... I can't see the change on this code review....

Reply:
Sorry. Now I fixed at 102cb3a